packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -65,87 +68,96 @@ Requires:       procps
+Provides:	bundled(golang(github.com/docker/docker/api/types/container))
+Provides:	bundled(golang(github.com/docker/docker/api/types/filters))
+Provides:	bundled(golang(github.com/docker/docker/api/types/mount))
+Provides:	bundled(golang(github.com/docker/docker/api/types/network))
+Provides:	bundled(golang(github.com/docker/docker/api/types/registry))
+Provides:	bundled(golang(github.com/docker/docker/api/types/strslice))
+Provides:	bundled(golang(github.com/docker/docker/api/types/swarm))
+Provides:	bundled(golang(github.com/docker/docker/api/types/versions))
+Provides:	bundled(golang(github.com/docker/docker/opts))
+Provides:	bundled(golang(github.com/docker/docker/pkg/archive))
+Provides:	bundled(golang(github.com/docker/docker/pkg/fileutils))
+Provides:	bundled(golang(github.com/docker/docker/pkg/homedir))
+Provides:	bundled(golang(github.com/docker/docker/pkg/idtools))
+Provides:	bundled(golang(github.com/docker/docker/pkg/ioutils))
+Provides:	bundled(golang(github.com/docker/docker/pkg/jsonlog))
+Provides:	bundled(golang(github.com/docker/docker/pkg/jsonmessage))
+Provides:	bundled(golang(github.com/docker/docker/pkg/longpath))
+Provides:	bundled(golang(github.com/docker/docker/pkg/mount))
+Provides:	bundled(golang(github.com/docker/docker/pkg/pools))
+Provides:	bundled(golang(github.com/docker/docker/pkg/promise))
+Provides:	bundled(golang(github.com/docker/docker/pkg/stdcopy))
+Provides:	bundled(golang(github.com/docker/docker/pkg/system))
+Provides:	bundled(golang(github.com/docker/docker/pkg/term))
+Provides:	bundled(golang(github.com/docker/docker/pkg/term/windows))
+Provides:	bundled(golang(github.com/docker/go-connections/nat))
+Provides:	bundled(golang(github.com/docker/go-connections/sockets))
+Provides:	bundled(golang(github.com/docker/go-plugins-helpers/sdk))
+Provides:	bundled(golang(github.com/docker/go-plugins-helpers/volume))
+Provides:	bundled(golang(github.com/docker/go-units))
+Provides:	bundled(golang(github.com/fsouza/go-dockerclient))
+Provides:	bundled(golang(github.com/go-ini/ini))
+Provides:	bundled(golang(github.com/golang/mock/gomock))
+Provides:	bundled(golang(github.com/jmespath/go-jmespath))
+Provides:	bundled(golang(github.com/opencontainers/go-digest))
+Provides:	bundled(golang(github.com/opencontainers/image-spec/specs-go))
+Provides:	bundled(golang(github.com/opencontainers/image-spec/specs-go/v1))
+Provides:	bundled(golang(github.com/opencontainers/runc/libcontainer/system))
+Provides:	bundled(golang(github.com/opencontainers/runc/libcontainer/user))
+Provides:	bundled(golang(github.com/pkg/errors))
+Provides:	bundled(golang(github.com/pmezard/go-difflib/difflib))
+Provides:	bundled(golang(github.com/stretchr/testify/assert))
+Provides:	bundled(golang(golang.org/x/net/context))
+Provides:	bundled(golang(golang.org/x/net/context/ctxhttp))
+Provides:	bundled(golang(golang.org/x/net/proxy))
+Provides:	bundled(golang(golang.org/x/sys/unix))
+Provides:	bundled(golang(golang.org/x/sys/windows))
```

Comment:
? do we need to change the formatting here?

Reply:
Nope, I will revert the formatting changes.